doc/cited.rst

```diff
@@ -0,0 +1,141 @@
+.. _cited
+
+Citations
```

Comment:
Can you use a more explicit name?

Publications from MNE users

see as an example http://neuroimage.usc.edu/brainstorm/Pub